CHANGELOG.md

```diff
@@ -1,3 +1,101 @@
+# 23.04 (2023-04-30)
+
+23.04 is a smaller release focusing on fixes. There were 77 contributors in this release. Thank you all!
+
+Features:
+
+- Add a config option to exclude declaration from LSP references request ([#6886](https://github.com/helix-editor/helix/pull/6886))
+- Enable injecting languages based on their file extension and shebang ([#3970](https://github.com/helix-editor/helix/pull/3970))
+- Sort the buffer picker by most recent access ([#2980](https://github.com/helix-editor/helix/pull/2980))
+
+Commands:
+
+- `:update` is now aliased as `:u` ([#6835](https://github.com/helix-editor/helix/pull/6835))
+- Add `extend_to_first_nonwhitespace` which acts the same as `goto_first_nonwhitespace` but always extends ([#6837](https://github.com/helix-editor/helix/pull/6837))
+- Add `:clear-register` for clearing the given register or all registers ([#5695](https://github.com/helix-editor/helix/pull/5695))
+
+Fixes:
+
+- Normalize LSP workspace paths ([#6517](https://github.com/helix-editor/helix/pull/6517))
+- Robustly handle invalid LSP ranges ([#6512](https://github.com/helix-editor/helix/pull/6512))
+- Fix line number display for LSP goto pickers ([#6559](https://github.com/helix-editor/helix/pull/6559))
+- Fix toggling of `soft-wrap.enable` option ([#6656](https://github.com/helix-editor/helix/pull/6656), [58e457a](https://github.com/helix-editor/helix/commit/58e457a), [#6742](https://github.com/helix-editor/helix/pull/6742))
+- Handle `workspace/configuration` requests from stopped language servers ([#6693](https://github.com/helix-editor/helix/pull/6693))
+- Fix possible crash from opening the jumplist picker ([#6672](https://github.com/helix-editor/helix/pull/6672))
+- Fix theme preview returning to current theme on line and word deletions ([#6694](https://github.com/helix-editor/helix/pull/6694))
+- Re-run crate build scripts on changes to revision and grammar repositories ([#6743](https://github.com/helix-editor/helix/pull/6743))
+- Fix crash on opening from suspended state ([#6764](https://github.com/helix-editor/helix/pull/6764))
+- Fix unwrap bug in DAP ([#6786](https://github.com/helix-editor/helix/pull/6786))
+- Always build tree-sitter parsers with C++14 and C11 ([#6792](https://github.com/helix-editor/helix/pull/6792), [#6834](https://github.com/helix-editor/helix/pull/6834), [#6845](https://github.com/helix-editor/helix/pull/6845))
+- Exit with a non-zero statuscode when tree-sitter parser builds fail ([#6795](https://github.com/helix-editor/helix/pull/6795))
+- Flip symbol range in LSP goto commands ([#6794](https://github.com/helix-editor/helix/pull/6794))
+- Fix runtime toggling of the `mouse` option ([#6675](https://github.com/helix-editor/helix/pull/6675))
+- Fix panic in inlay hint computation when view anchor is out of bounds ([#6883](https://github.com/helix-editor/helix/pull/6883))
+- Change Git repository discovery to worktree-only ([#6890](https://github.com/helix-editor/helix/pull/6890))
```

Comment:
This is correct but a bit to technical for a changelog IMO. Instead I would describe what this actually improves (instead of how). For example:


```suggestion
- Significantly improve performance of git discovery on slow file systems ([#6890](https://github.com/helix-editor/helix/pull/6890))
```